deployment/conf/pkgserver.nginx.conf

```
@@ -8,6 +8,12 @@ ${LISTEN_BLOCK}
     # For our content-addressed resources, our happy path is to look in the cache
     # for the requested content:
     location ~ /(registry|package|artifact)/ {
+        # Everything served from here are gzipped tarballs but it is not
+        # recognized by nginx since the files don't have the .tar.gz file
+        # extension so we set the following headers manually.
+        add_header Content-Type application/tar;
+        add_header Content-Encoding gzip;
```

```suggestion
        add_header Content-Type application/x-gzip;
```

We don't actually want to set `Content-Encoding` here, because if we do, browsers may choose to unwrap that encoding.  I didn't know this in the past, but now I do; the `Content-Encoding` is primarily meant to refer to encoding that is done on-the-fly by webservers.  That's why it can be negotiated with `Accept-Encoding` request headers and whatnot.  We kind of play with this a little bit on the Storage Server end (we have support for serving `.zst` packages, for instance) but as it stands now, we aren't actually doing any of that on the Pkg server side.

---

Alright, I just mirrored what we set from PkgServer so I guess that should be updated too then?